### ISSUE TITLE
Build bitcoin module (all modules) with Gradle

### ISSUE THREAD

**Issue by msgilligan** [JavaMoney/javamoney-shelter#18]
So, I've been using a Gradle build within the `groovylang-support` directory and would be happy to convert the Maven-based build in the `bitcoin` module over to Gradle.

This would allow the whole "shelter" to be built with Gradle and would also get the `groovylang-support` modules hooked in to Travis.

If this is OK, I'll put together a Pull Request.

**Comment by keilw** [comment]
If it's more consistent to build the whole shelter, why not. Android also likes Gradle nowadays, so depending on what will be the final outcome of the Backport discussion (either separate modules or a variant of javax.money if we are able to provide a TCK for both RI and API) something for Android in shelter could also use it.

**Comment by msgilligan** [comment]
Yes, I think it would be better and more consistent if we use a single build tool for the whole shelter. And, yes, Gradle should be helpful in building and testing for Android.

I want to make sure that tests are run, artifacts are published, and API docs are generated, etc for all modules in the repo. I probably won't have time to work on this until early April, but may make some incremental steps (such as a44145383686128b73cdc6e3f86921f6fded09fe before then)
